### PR TITLE
add(option.series.graph):add auto curveness doc

### PR DESCRIPTION
**Pull request by wf123537200**
link to [12590](https://github.com/apache/incubator-echarts/pull/12590)

### CODE REVIEW

File: en/option/series/graph.md [hunk 1]
```diff
@@ -213,6 +213,15 @@ The label style of node in this category.
 #### label(Object)
 {{ use:partial-label(prefix="####") }}
 
+## autoCurveness(number|Array) = 20
+
+For the situation where there are multiple links between nodes, the curveness of each links is automatically calculated.
+
+When set to Number, it indicates the length of the edge curveness array between two points, and the calculation result is given by the internal algorithm.
```
Comment (susiwen8) [inline en/option/series/graph.md:220]: Change Number to `Number`

File: en/option/series/graph.md [hunk 2]
```diff
@@ -213,6 +213,15 @@ The label style of node in this category.
 #### label(Object)
 {{ use:partial-label(prefix="####") }}
 
+## autoCurveness(number|Array) = 20
+
+For the situation where there are multiple links between nodes, the curveness of each links is automatically calculated.
+
+When set to Number, it indicates the length of the edge curveness array between two points, and the calculation result is given by the internal algorithm.
+
+When set to Array, it means that the curveness array is directly specified, and the multilateral curvature is directly selected from the array.
```
Comment (susiwen8) [inline en/option/series/graph.md:222]: Change Array to `Array`

File: zh/option/series/graph.md [hunk 3]
```diff
@@ -261,6 +261,16 @@ edgeSymbol: ['circle', 'arrow']
 {{ use:partial-label(prefix="####") }}
 
 
+## autoCurveness(number|Array) = 20
+
+针对点之间存在多边的情况，自动计算各边曲率。
+
+设置为 Number时，表示两点间边曲率数组的长度，由内部算法给出计算结果。
+
+设置为 Array时, 表示直接指定曲率数组，多边曲率会从数组中直接按顺序选取。
```
Comment (susiwen8) [inline zh/option/series/graph.md:270]: Same here

File: zh/option/series/graph.md [hunk 4]
```diff
@@ -261,6 +261,16 @@ edgeSymbol: ['circle', 'arrow']
 {{ use:partial-label(prefix="####") }}
 
 
+## autoCurveness(number|Array) = 20
+
+针对点之间存在多边的情况，自动计算各边曲率。
```
Comment (plainheart) [inline zh/option/series/graph.md:266]: Hereafter, it's better to change `点` to `节点`.

File: en/option/series/graph.md [hunk 5]
```diff
@@ -213,6 +213,15 @@ The label style of node in this category.
 #### label(Object)
 {{ use:partial-label(prefix="####") }}
 
+## autoCurveness(number|Array) = 20
+
+For the situation where there are multiple links between nodes, the curveness of each links is automatically calculated.
+
+When set to `Number`, it indicates the length of the edge curveness array between two points, and the calculation result is given by the internal algorithm.
```
Comment (plainheart) [inline en/option/series/graph.md:220]: The same as Chinese, `points` might be `nodes`.

File: en/option/series/graph.md [hunk 6]
```diff
@@ -213,6 +213,15 @@ The label style of node in this category.
...
 
+## autoCurveness(number|Array) = 20
+
+For the situation where there are multiple links between nodes, the curveness of each links is automatically calculated.
+
+When set to `Number`, it indicates the length of the edge curveness array between two points, and the calculation result is given by the internal algorithm.
+
+When set to `Array`, it means that the curveness array is directly specified, and the multilateral curvature is directly selected from the array.
+
+**Notice：** if [lineStyle.curveness](~series-graph.lineStyle.curveness) has been setted, this property is invalid.
```
Comment (plainheart) [inline en/option/series/graph.md:224]: I'm afraid there is no such usage of `setted`, the past participle of `set` is still `set`.

File: en/option/series/graph.md [hunk 7]
```diff
@@ -213,6 +213,15 @@ The label style of node in this category.
 #### label(Object)
 {{ use:partial-label(prefix="####") }}
 
+## autoCurveness(number|Array) = 20
+
+For the situation where there are multiple links between nodes, the curveness of each links is automatically calculated.
+
+When set to `Number`, it indicates the length of the edge curveness array between two points, and the calculation result is given by the internal algorithm.
+
+When set to `Array`, it means that the curveness array is directly specified, and the multilateral curvature is directly selected from the array.
```
Comment (plainheart) [inline en/option/series/graph.md:222]: Is `curveness` the same as `curvature`?

File: en/option/series/graph.md [hunk 8]
```diff
@@ -213,6 +213,15 @@ The label style of node in this category.
 #### label(Object)
 {{ use:partial-label(prefix="####") }}
 
+## autoCurveness(number|Array) = 20
+
+For the situation where there are multiple links between nodes, the curveness of each links is automatically calculated.
```
Comment (plainheart) [inline en/option/series/graph.md:218]: `links` should be `link`.

File: zh/option/series/graph.md [hunk 9]
```diff
@@ -261,6 +261,16 @@ edgeSymbol: ['circle', 'arrow']
...
+
+设置为 `Number` 时，表示两点间边曲率数组的长度，由内部算法给出计算结果。
+
+设置为 `Array` 时, 表示直接指定曲率数组，多边曲率会从数组中直接按顺序选取。
```
Comment (plainheart) [inline zh/option/series/graph.md:270]: `,` should be `，`

**Comment by wf123537200** [comment]
@plainheart  thank you for review，the changes requested has been fixed.

File: zh/option/series/graph.md [hunk 10]
```diff
@@ -261,6 +261,16 @@ edgeSymbol: ['circle', 'arrow']
...
 
+## autoCurveness(number|Array) = 20
+
+针对节点之间存在多边的情况，自动计算各边曲率。
+
+设置为 `Number` 时，表示两点间边曲率数组的长度，由内部算法给出计算结果。
+
+设置为 `Array` 时，表示直接指定曲率数组，多边曲率会从数组中直接按顺序选取。
+
+**注意：** 如果设置[lineStyle.curveness](~series-graph.lineStyle.curveness)则此属性失效。
```
Comment (Ovilia) [inline zh/option/series/graph.md:272]: 中文混排需要加入空格，这里“设置”后，“则”前应该加个空格

**Comment by Ovilia** [comment]
Please link the PR of ECharts changes to this.
Thanks!

File: en/option/series/graph.md [hunk 11]
```diff
@@ -213,6 +213,15 @@ The label style of node in this category.
 #### label(Object)
 {{ use:partial-label(prefix="####") }}
 
+## autoCurveness(number|Array) = 20
+
+For the situation where there are multiple links between nodes, the curveness of each link is automatically calculated.
+
+When set to `Number`, it indicates the length of the edge curveness array between two nodes, and the calculation result is given by the internal algorithm.
```
Comment (Ovilia) [inline en/option/series/graph.md:220]: `Number` should be `number` since it's primitive.

**Comment by Ovilia** [comment]
@wf123537200 Sorry I found another small mistake.
It would be better if you include `apache/incubator-echarts#12590` rather than `#12590` next time. I will squash this PR later.
Thanks!

**Pull request merged**